aireview/src/aireview/domain/services/code_analyzer.py

```diff
@@ -32,4 +32,8 @@ def _build_analysis_prompt(self, file: PullRequestFile):
 
 
     def _parse_analysis(self, analysis):
-        return ReviewComment.from_json(analysis)
\ No newline at end of file
+        review_comments: [ReviewComment] = []
```

Comment:
**style**: Type hint for review_comments should use List from typing module instead of [] syntax for better type safety